storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh

```diff
@@ -0,0 +1,589 @@
+    fi
+    cd $WORKDIR
+    rm -fr rpmbuild
+    mkdir -vp rpmbuild/{SOURCES,SPECS,BUILD,SRPMS,RPMS}
+    cp $SRC_RPM rpmbuild/SRPMS/
+    #
+    echo "RHEL=${RHEL}" >> percona-xtrabackup-8.0.properties
+    echo "ARCH=${ARCH}" >> percona-xtrabackup-8.0.properties
+    #
+    SRCRPM=$(basename $(find . -name '*.src.rpm' | sort | tail -n1))
+
+    if [ "${RHEL}" -eq 8 ]; then
+        export PATH="/usr/local/bin:$PATH"
+    elif [ "${RHEL}" -eq 7 ]; then
+        source /opt/rh/devtoolset-7/enable
+    elif [ "${RHEL}" -eq 6 ]; then
+        source /opt/rh/devtoolset-7/enable
+        source /opt/rh/rh-python36/enable
+    fi
+
+    rpmbuild --define "_topdir ${WORKDIR}/rpmbuild" --define "dist .el${RHEL}" --rebuild rpmbuild/SRPMS/${SRCRPM}
+    return_code=$?
+    if [ $return_code != 0 ]; then
+        exit $return_code
+    fi
+    mkdir -p ${WORKDIR}/rpm
+    mkdir -p ${CURDIR}/rpm
+    cp rpmbuild/RPMS/*/*.rpm ${WORKDIR}/rpm
+    cp rpmbuild/RPMS/*/*.rpm ${CURDIR}/rpm    
+}
+
+build_source_deb(){
+    if [ $SDEB = 0 ]
+    then
+        echo "source deb package will not be created"
+        return;
+    fi
+    if [ "x$OS" = "xrmp" ]
+    then
+        echo "It is not possible to build source deb here"
+        exit 1
+    fi
+    rm -rf percona-xtrabackup*
+    get_tar "source_tarball"
+    rm -f *.dsc *.orig.tar.gz *.debian.tar* *.changes
+
+    TARFILE=$(basename $(find . -name 'percona-xtrabackup-*.tar.gz' | sort | tail -n1))
+    NAME=$(echo ${TARFILE}| awk -F '-' '{print $1"-"$2}')
+    VERSION=$(echo ${TARFILE%.tar.gz} | awk -F '-' '{print $3}')
+    SHORTVER=$(echo ${VERSION} | awk -F '.' '{print $1"."$2}')
+
+    echo "DEB_RELEASE=${DEB_RELEASE}" >> percona-xtrabackup-8.0.properties
+
+    NEWTAR=${NAME}-80_${VERSION}.orig.tar.gz
+    mv ${TARFILE} ${NEWTAR}
+
+    tar xzf ${NEWTAR}
+    cd percona-xtrabackup-$VERSION
+    cp -av storage/innobase/xtrabackup/utils/debian .
+    dch -D unstable --force-distribution -v "${VERSION}-${DEB_RELEASE}" "Update to new upstream release Percona XtraBackup ${VERSION}"
+    dpkg-buildpackage -S
+
+    cd ${WORKDIR}
+
+    mkdir -p $WORKDIR/source_deb
+    mkdir -p $CURDIR/source_deb
+
+    cp *.dsc $WORKDIR/source_deb
+    cp *.orig.tar.gz $WORKDIR/source_deb
+    cp *.debian.tar.* $WORKDIR/source_deb
+    cp *.changes $WORKDIR/source_deb
+    cp *.dsc $CURDIR/source_deb
+    cp *.orig.tar.gz $CURDIR/source_deb
+    cp *.debian.tar.* $CURDIR/source_deb
+    cp *.changes $CURDIR/source_deb
+    
+}
+
+build_deb(){
+    if [ $DEB = 0 ]
+    then
+        echo "source deb package will not be created"
+        return;
+    fi
+    if [ "x$OS" = "xrmp" ]
+    then
+        echo "It is not possible to build source deb here"
+        exit 1
+    fi
+    for file in 'dsc' 'orig.tar.gz' 'debian.tar.*' 'changes'
+    do
+        get_deb_sources $file
+    done
+    cd $WORKDIR
+
+    echo "DEB_RELEASE=${DEB_RELEASE}" >> percona-xtrabackup-8.0.properties
+    DEBIAN_VERSION="$(lsb_release -sc)"
```

Comment:
you should already have value in ${OS_NAME}

Reply:
Done